docs/platforms/javascript/common/index.mdx

```diff
@@ -95,15 +100,9 @@ Sentry captures data by using an SDK within your application’s runtime.
 
 ## Configure
 
-<PlatformSection notSupported={["javascript.nextjs", "javascript.astro"]}>
-
-Configuration should happen as early as possible in your application's lifecycle.
-
-</PlatformSection>
-
```

Comment:
<PlatformSection notSupported={["javascript.nextjs", "javascript.astro"]}>

Configuration should happen as early as possible in your application's lifecycle.

</PlatformSection>